src/commands/deploy/internal/executeNonSasScript.ts

```diff
@@ -8,9 +8,10 @@ import { isShellScript, isPowerShellScript } from '../../../utils/file'
 export async function executeNonSasScript(scriptPath: string) {
   const fileExtension = path.extname(scriptPath)
 
+  const { buildDestinationResultsLogsFolder } = process.sasjsConstants
```

Comment:
would be good to get a comment here explaining what `buildDestinationResultsLogsFolder` is.  The variable name contains both "build" and "results" so it might be ambiguous as to whether it's `sasjsbuild` or `sasjsresults`

Reply:
good point!
For now, we can add a comment explaining this point but we should rename these constants in the future.